lib/block.ml

```diff
@@ -300,6 +301,37 @@ let required_packages t =
   in
   List.map f (get_labels t "require-package")
 
+let require_re =
+  let open Re in
+  seq [str "#require \""; group (rep1 any); str "\""]
+
+let require_from_line line =
+  let open Util.Result.Infix in
+  let re = Re.compile require_re in
+  match Re.exec_opt re line with
+  | None -> Ok Library.Set.empty
+  | Some group ->
+    let matched = Re.Group.get group 1 in
+    let libs_str = String.cuts ~sep:"," matched in
+    Util.Result.List.map ~f:Library.from_string libs_str >>| fun libs ->
+    List.fold_left (fun acc l -> Library.Set.add l acc) Library.Set.empty libs
+
+let require_from_lines lines =
+  let open Util.Result.Infix in
+  Util.Result.List.map ~f:require_from_line lines >>| fun libs ->
+  List.fold_left Library.Set.union Library.Set.empty libs
+
+let required_libraries = function
+  | { value = Toplevel _; contents; _} -> require_from_lines contents
+  | { value = (Raw | OCaml | Error _ | Cram _); _ } -> Ok Library.Set.empty
+
+let required_packages t =
```

Comment:
This function is not used.
It's not useful to keep it to not break the API because its type change and it doesn't do the same thing.